src/DetailsView/components/issues-view.tsx

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { ISelection } from 'office-ui-fabric-react/lib/DetailsList';
+import * as React from 'react';
+
+import { VisualizationConfiguration } from '../../common/configs/visualization-configuration';
+import { VisualizationConfigurationFactory } from '../../common/configs/visualization-configuration-factory';
+import { NamedSFC } from '../../common/react/named-sfc';
+import { FeatureFlagStoreData } from '../../common/types/store-data/feature-flag-store-data';
+import { TabStoreData } from '../../common/types/store-data/tab-store-data';
+import { UserConfigurationStoreData } from '../../common/types/store-data/user-configuration-store';
+import { VisualizationScanResultData } from '../../common/types/store-data/visualization-scan-result-data';
+import { VisualizationStoreData } from '../../common/types/store-data/visualization-store-data';
+import { VisualizationType } from '../../common/types/visualization-type';
+import { DetailsViewToggleClickHandlerFactory } from '../handlers/details-view-toggle-click-handler-factory';
+import { IssuesTable, IssuesTableDeps } from './issues-table';
+import { IssuesTableHandler } from './issues-table-handler';
+
+export type IssuesViewDeps = IssuesTableDeps;
+
+export interface IssuesViewProps {
+    deps: IssuesViewDeps;
+    tabStoreData: TabStoreData;
+    featureFlagStoreData: FeatureFlagStoreData;
+    selectedTest: VisualizationType;
+    visualizationStoreData: VisualizationStoreData;
+    visualizationScanResultData: VisualizationScanResultData;
+    visualizationConfigurationFactory: VisualizationConfigurationFactory;
+    clickHandlerFactory: DetailsViewToggleClickHandlerFactory;
+    issuesSelection: ISelection;
+    issuesTableHandler: IssuesTableHandler;
+    configuration: VisualizationConfiguration;
+    userConfigurationStoreData: UserConfigurationStoreData;
+}
+
+export const IssuesView = NamedSFC<IssuesViewProps>('IssuesView', ({ children, ...props }) => {
```

Comment:
Not sure I like the name 'IssuesView' as it's broad and could still even encompass card semantically.

I'd like to call this something like DetailsListIssuesView or TableIssuesView, which might be more relevant.